.github/workflows/mergebot.yml

```diff
@@ -0,0 +1,23 @@
+name: Mergebot
+
+on:
+  workflow_run:
+    workflows:
+      - Pull request
+    types:
+      - completed
+  workflow_dispatch:
+
+jobs:
+  auto-merge:
+    name: Auto merge
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
```

Comment:
We can bump this up if needed, runs in public repos are free

Reply:
It's not a matter of cost. As the action runs usually for 20 seconds or less, it timing out after a minute will be an indication of something going wrong. I'd rather keep it as is.